backends/advanced/src/advanced_omi_backend/services/transcription/__init__.py

```
@@ -196,24 +239,72 @@ async def transcribe(self, audio_data: bytes, sample_rate: int, diarize: bool =
+                                    event = json.loads(line)
+                                    if (
+                                        event.get("type") == "progress"
+                                        and progress_callback
+                                    ):
+                                        progress_callback(event)
+                                    elif event.get("type") == "result":
+                                        data = event
+                                if data is None:
+                                    raise RuntimeError(
+                                        f"NDJSON stream from '{self._name}' ended without a result event"
+                                    )
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)